### PR TITLE
Re-enable HKH region

### PR DESCRIPTION
**Pull request by jhkennedy**
This:
*  adds flood_depth maps to the product packages
* updates the search for this years monsoon season

### CODE REVIEW

**Comment by jhkennedy** [comment]
@hjkristenson do you know which flood depth GeoTIFFs we'll want to transfer? I haven't added that yet.

**Comment by jhkennedy** [comment]
@hjkristenson also, should the be transferred to the same `HKHwatermaps` prefix?

**Comment by hjkristenson** [comment]
> @hjkristenson do you know which flood depth GeoTIFFs we'll want to transfer? I haven't added that yet.

I'm not at all familiar with the flood depth products. I think @jacquelynsmale was going to look into it.

**Comment by hjkristenson** [comment]
> @hjkristenson also, should the be transferred to the same `HKHwatermaps` prefix?

I'm going to post in the HydroSAR Slack to see how they want to deal with this. If they're ready to delete the old data, we can just use the same prefix as before.

**Comment by jacquelynsmale** [comment]
> > @hjkristenson do you know which flood depth GeoTIFFs we'll want to transfer? I haven't added that yet.
> 
> I'm not at all familiar with the flood depth products. I think @jacquelynsmale was going to look into it.
> 

I believe you will want the _FloodDepth.tif option. I am going to confirm with the HydroSAR team what each GeoTIFF's purpose is, but we've been comparing these data.

**Comment by jhkennedy** [comment]
@jacquelynsmale @hjkristenson sounds good -- I'm interested in both:
* what data do they want to preserve in the bucket
* what do they want to show up in the image services.

**Review by hjkristenson — COMMENTED** [review]
The changes look good to me. Once we know what products need to be transferred, that can be updated in line 33 of the JSON.

Do we need to be adjusting the parameters in the JSON files for the other services, too? Or do those changes only impact the flood depth products?

**Comment by jhkennedy** [comment]
> Do we need to be adjusting the parameters in the JSON files for the other services, too? Or do those changes only impact the flood depth products?

Shouldn't need too -- the parameters are just related to flood depth, and only used if `"include_flood_depth": true` which is `false` by default.

**Comment by jhkennedy** [comment]
Unless, or course, we want flood depth included in other services

File: image_server/nasa_disasters_hkh/batchFiles/update_hkh.py [hunk 1]
```diff
@@ -2,11 +2,12 @@
 # then runs them in sequence
 
 import os
```
Comment (asjohnston-asf) [inline image_server/nasa_disasters_hkh/batchFiles/update_hkh.py:4]:
```suggestion
import datetime
import os
```

**Review by asjohnston-asf — APPROVED** [review]

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**